presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -525,6 +526,11 @@ public SystemSessionProperties(
                         "Experimental: Enable statistics calculator",
                         featuresConfig.isEnableStatsCalculator(),
                         false),
+                integerProperty(
+                        MAX_TASKS_PER_STAGE,
+                        "Limits the number of tasks for a fixed distribution stage",
```

Comment:
nit: `Limit...`? or we could just say `Maximum number of tasks for a fixed distribution stage`.

Reply:
Changed to `Maximum number of tasks for a non source distributed stage`